common-arcana.lic

```diff
@@ -303,7 +303,10 @@ module DRCA
     observed_planets = []
 
     planets.each do |planet|
-      DRC.bput("center telescope on #{planet}", 'You put your eye')
+      case DRC.bput("center telescope on #{planet}", 'You put your eye', 'The pain is too much')
+      when 'The pain is too much'
+        return observed_planets
```

Comment:
Does this need to stow the telescope before returning? (see line 315)

Reply:
Yes good call. I tested this in combat-trainer, and it must have figured out that it needed to stow it.

Reply:
I will just break out of the for loop then